data/org.freedesktop.portal.ScreenCast.xml

```diff
@@ -260,6 +260,13 @@
               This property was added in version 3 of this interface.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>output_names</term>
+            <listitem><para>
+            If a monitor is being screencasted, then this property contains the
+            wayland output name. Only available for monitor streams.
```

Comment:
The indentation seems a bit off; also, in some way, we should reference what the name corresponds to. By no means are anything directly tied to Wayland here, the screen cast API can just as well be used on an X11 session. Thus, we should probably list at least those two. For X11, I guess the xrandr name is fine. For Wayland, it'd be better to be a bit more specific, since "wayland output name" is a bit ambiguous.

Reply:
Thanks for the comment.

- Adjusted the indentation.
- Generalized the description to mention both X11 and Wayland.
- Added more specific details about what this string would contain in context of Wayland.